### DISCUSSION TITLE
How to specify where the doc/ folder is located relative to mkdocs.yml

### DISCUSSION THREAD

**Discussion by sfc-gh-dwilczak** [squidfunk/mkdocs-material#7430] [Q&A] [answered]
Hey Team!

How would I go about specifying where the doc/ folder is located in the mkdocs.yaml file?

In my scenario:

- mkdocs.yml 
- src/
- - docs/
- - mkdocs.yml

The reason for this is that the ``src/`` is used in an externally facing application, where many of the doc functionalities and pages are different, but I would like to use some of the markdown files in public-facing docs in gh-pages. Essentially my goal is to have two docs but both use the same ``docs/`` folder.

**Comment by alexvoss** [comment, marked as answer]
The [docs_dir](https://www.mkdocs.org/user-guide/configuration/#docs_dir) configuration option should help with this.

**Reply by sfc-gh-dwilczak** [reply]
Thank you for this! It's been so long since I had to read mkdocs documentation that I completely forgot that is the base for mkdocs-material.
